tests/unit/SummaryRestoreBusiness.spec.ts

```diff
@@ -14,52 +14,193 @@ Vue.use(Vuelidate)
+  afterEach(() => {
     wrapper.destroy()
   })
 
-  it('shows full restoration details with Applicant Relationship=Officer', () => {
-    const wrapper = wrapperFactory()
-    expect(wrapper.find('#applicants-relationships').text()).toContain('Applicant\'s relationship(s): Officer.')
-    wrapper.destroy()
+  it('does not show errors if step is valid', () => {
+    expectSummaryBusinessRestoreExists()
+    expectSectionValid()
+    expectErrorMessage(false)
+  })
+
+  it('shows "step is unfinished" message if Business Name is invalid', async () => {
+    store.state.stateModel.restoration.businessNameValid = false
+    await Vue.nextTick()
+    expectSummaryBusinessRestoreExists()
+    expectSectionValid(false)
+    expectErrorMessage()
+  })
+
+  it('shows "step is unfinished" message if Name Translations is invalid', async () => {
+    store.state.stateModel.nameTranslationsValid = false
+    await Vue.nextTick()
+    expectSummaryBusinessRestoreExists()
+    expectSectionValid(false)
+    expectErrorMessage()
+  })
+
+  it('shows "step is unfinished" message if Restoration Type is invalid', async () => {
+    store.state.stateModel.restoration.restorationTypeValid = false
+    await Vue.nextTick()
+    expectSummaryBusinessRestoreExists()
+    expectSectionValid(false)
+    expectErrorMessage()
+  })
+
+  it('shows "step is unfinished" message if Approval Type is invalid', async () => {
+    store.state.stateModel.restoration.approvalTypeValid = false
+    await Vue.nextTick()
+    expectSummaryBusinessRestoreExists()
+    expectSectionValid(false)
+    expectErrorMessage()
+  })
+
+  it('shows NR company name and description', () => {
+    expectSummaryBusinessRestoreExists()
+    expectSectionValid()
+    expectErrorMessage(false)
+    expect(wrapper.findAll('article:nth-of-type(1) label').at(0).text()).toBe('Name')
+    expect(wrapper.find('#company-name').text()).toBe('NR Approved Name')
+    expect(wrapper.find('#company-description').text()).toBe('BC Limited Company')
+  })
+
+  it('shows numbered company name', async () => {
+    store.state.stateModel.correctNameOption = 'correct-name-to-number'
+    store.state.stateModel.nameRequestApprovedName = '1234567 B.C. LTD.'
+    await Vue.nextTick()
+    expectSummaryBusinessRestoreExists()
+    expectSectionValid()
+    expectErrorMessage(false)
+    expect(wrapper.find('#company-name').text()).toBe('1234567 B.C. LTD.')
+  })
+
+  it('shows unknown company name', async () => {
+    store.state.stateModel.correctNameOption = null
+    await Vue.nextTick()
+    expectSummaryBusinessRestoreExists()
+    expectSectionValid()
+    expectErrorMessage(false)
+    expect(wrapper.find('#company-name').text()).toBe('[Unknown]')
+  })
+
+  it('shows name translations', () => {
+    expectSummaryBusinessRestoreExists()
+    expectSectionValid()
+    expectErrorMessage(false)
+    expect(wrapper.findAll('article:nth-of-type(1) label').at(1).text()).toBe('Name Translation')
```

Comment:
Note the selector to find the first sibling article.

I couldn't do that for the labels because they're not siblings (they have different parents).